packages/react-dom-bindings/src/events/plugins/FormActionEventPlugin.js

```diff
@@ -60,10 +111,6 @@ function extractEvents(
     }
   }
```

Comment:
The submitterAction that's coming from `formAction` needs to be coerced too. Technically it was wrong before because it wasn't ignoring boolean/symbol and was treating them as the submitter taking over.

Reply:
Also the submitter being set to `null` in that branch is not correct in the case where it's not a function. It previously just assumed that if it wasn't a function it would've exited early anyway.